README.md

```diff
@@ -1,58 +1,85 @@
-# provider-kraftcloud
+# provider-unikraft-cloud
 
-`provider-kraftcloud` is an early but functional [Crossplane](https://crossplane.io/) Provider for the Kraftcloud API. This provider allows you to manage Kraftcloud resources as managed resources within Crossplane. 
+`provider-unikraft-cloud` is an early but functional [Crossplane](https://crossplane.io/) Provider for the Unikraft Cloud API. This provider allows you to manage Unikraft Cloud resources as managed resources within Crossplane.
 
-This initial version supports the `Instance` resource, allowing you to manage your Kraftcloud instances seamlessly.
+This initial version supports the `Instance` resource, allowing you to manage your Unikraft Cloud instances seamlessly.
 
 ## Supported Resources
 
 ### Instance
 
-The `Instance` type allows you to specify the desired state of your Kraftcloud instance.
+The `Instance` type allows you to specify the desired state of your Unikraft Cloud instance.
 
 Sample resource:
 
 ```yaml
-apiVersion: compute.kraftcloud.crossplane.io/v1alpha1
+apiVersion: compute.unikraft-cloud.crossplane.io/v1alpha1
```

Comment:
Can you please change the API to be:
```
cloud.unikraft.crossplane.io
```
This will be then:
```
compute.cloud.unikraft.crossplane.io/v1alpha1
```
Thanks!

Reply:
So everything `unikraft-cloud` in yaml becomes `cloud.unikraft`, oke